Source/Analytics/Events.Reporting/CaseReports/CaseReportReceived.cs

```diff
@@ -1,31 +1,17 @@
 /*---------------------------------------------------------------------------------------------
- *  Copyright (c) 2017 International Federation of Red Cross. All rights reserved.
+ *  Copyright (c) The International Federation of Red Cross and Red Crescent Societies. All rights reserved.
  *  Licensed under the MIT License. See LICENSE in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
+
 using System;
+using Dolittle.Artifacts;
 using Dolittle.Events;
 
 namespace Events.Reporting.CaseReports
 {
+    [Artifact("58a3dbf3-bcd0-4722-b21b-65d06558df53", 1)]
     public class CaseReportReceived : IEvent
     {
-        public CaseReportReceived(Guid dataCollectorId, Guid healthRiskId,
-            string origin, string message, int numberOfMalesUnder5, int numberOfMalesAged5AndOlder,
-            int numberOfFemalesUnder5, int numberOfFemalesAged5AndOlder, double longitude,
-            double latitude, DateTimeOffset timestamp)
-        {
-            this.DataCollectorId = dataCollectorId;
-            this.HealthRiskId = healthRiskId;
-            this.Origin = origin;
-            this.Message = message;
-            this.NumberOfMalesUnder5 = numberOfMalesUnder5;
-            this.NumberOfMalesAged5AndOlder = numberOfMalesAged5AndOlder;
-            this.NumberOfFemalesUnder5 = numberOfFemalesUnder5;
-            this.NumberOfFemalesAged5AndOlder = numberOfFemalesAged5AndOlder;
-            this.Longitude = longitude;
-            this.Latitude = latitude;
-            this.Timestamp = timestamp;
-        }
         public Guid DataCollectorId { get; }
         public Guid HealthRiskId { get; }
```

Comment:
Should this be a HealthRiskId and not a Guid?

Reply:
Can't remember the exact reason why, but for Event-types, we use primitive types and not concepts... So in this case.